src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -3,21 +3,36 @@ layout: article
 title: Non-interactive
 description: Deploy changes to Appwrite projects to migrate databases and collections schema, functions, teams, buckets, and more.
 ---
-The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server. You can enable the non-interactive mode for the Appwrite CLI by setting the `project ID`, `endpoint`, and `API Key`:
+
+The [Appwrite CLI](/docs/tooling/command-line/installation) can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server.
+
+# Authentication {% #authentication %}
+
+In non-interactive mode, you can authenticate using two methods: email and password or API keys. While using an [API key](/docs/advanced/platform/api-keys) does not require an email and password to authenticate, the API key needs to have its [scopes](/docs/advanced/platform/api-keys#scopes) configured and the API key does not work for project API calls that update project settings.
```

Comment:
Regarding API key limitation, we can say that administrative operations cannot be done such as changing organization billing, deleting a project, or creating a new project. For those you must email&password login